pt/python_introduction/README.md

```diff
@@ -924,14 +924,14 @@ Ainnda se lembra das listas? Vamos fazer uma lista de garotas:
 
```

Comment:
Corrigir 'Ainnda' para 'Ainda'

Reply:
Same thing. She might want to help improving the translation, but if she left a few things out that were already there, it doesn't matter. It shouldn't stop us from approving this PR.